modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js

```diff
@@ -183,7 +193,7 @@ export default class Tile3DLayer extends CompositeLayer {
     );
   }
 
-  _create3DModelTileLayer(tileHeader) {
+  _get3DModelLayer(tileHeader) {
```

Comment:
Nit: I think `create` or `make` is better than `get` for functions that created something new. `get` seems like a cheap accessor.